metron-platform/metron-common/src/main/java/org/apache/metron/common/configuration/ConfigurationsUtils.java

```diff
@@ -343,25 +343,42 @@ public static void uploadConfigsToZookeeper(String rootFilePath, CuratorFramewor
    * @param type config type to upload configs for
    * @param configName specific config under the specified config type
    */
-  public static void uploadConfigsToZookeeper(String rootFilePath, CuratorFramework client,
-      ConfigurationType type, Optional<String> configName) throws Exception {
+  public static void uploadConfigsToZookeeper(
+          String rootFilePath,
```

Comment:
When reading from file, the API should explicitly require file Paths as opposed to just strings.
It makes the api methods easier to understand when there are a bunch of similar methods.

Reply:
Sure, but this method signature already existed.  Do I need to fix that on this PR?

Reply:
No.  I guess I miss read the ambition of this pr, given the meta-pr and all ;)

Reply:
Yeah I get it. ;). It wasn't my preference to include all of that here, but I bend to make people happy.

Reply:
I am familiar with that ;)

Reply:
I'll open a jira

Reply:
METRON-1342